### ISSUE TITLE
NTR: Glabellar skin wrinkled (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8013]
New term request Glabellar skin wrinkled (elthat_180619044835): Rhytides (less deep than wrinkles)  UMLS: 1996997

**Comment by pnrobinson** [comment]
@MickeySegal
After a certain age, rhytides are a normal finding. Can you specify?

**Comment by MickeySegal** [comment]
This was meant to describe things like https://www.omim.org/entry/211380 "Skin wrinkling over the glabellar region seems common".  We include it in normals but at a much later age than in such diseases.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000744
- label: Premature glabellar skin wrinkling
- parents: [Abnormality of the glabella (HP:0002056)](https://hpo.jax.org/app/browse/term/HP:0002056) | [Premature skin wrinkling (HP:0100678)](https://hpo.jax.org/app/browse/term/HP:0100678)
- pmid: PMID:20949527
- def: The presence in the skin of the glabellar region of an increased degree of wrinkling (irregular folds and indentations) of the skin as compared with age-related norms.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8013
- synonyms:
